SEED-platform/seed
# fix: fix delete cycles flow

#### Any background context you want to provide?
Ability to delete cycles from org admin page is broken.

#### What's this PR do?
Fixes deletion of cycles.

#### How should this be manually tested?
1. Create two organizations
2. go through cycle deletion flow as described in [this PR](https://github.com/SEED-platform/seed/pull/2568) when your default org (ie the dropdown in upper right) is the same as the one for the cycle you're deleting (ie the cycle is part of your default org), and then again when deleting a cycle not part of your default org. It should work for both cases.

When deleting cycles from non-default org, clicking the "view properties" button in the deletion modal should open a new tab with the correct cycle and organization set. In addition, it should refresh the cycles settings page (ie the page the deletion modal is on), which should then have the default org (drop down in upper right) set to the organization of the cycle you're deleting.

Test celery by uploading a file that has 100s of lines in it (I just copy/pasted first few lines in the example-pm properties file) and verify that the progress bar works as expected.
 
#### What are the relevant tickets?
#2629
